### DISCUSSION TITLE
Update soon?

### DISCUSSION THREAD

**Discussion by patc3** [zim-desktop-wiki/zim-desktop-wiki#2676] [General]
Hello,

Is there an update in the pipeline soon, or should I instead look into installing from github? (I've never installed from github before, that's why I ask before attempting it)

Thanks a lot

**Comment by jaap-karssenberg** [comment]
I haven't planned a new release yet, but I could. What fix are you looking
for?

Op do 12 sep. 2024 15:36 schreef Patrick Coulombe ***@***.***

**Comment by patc3** [comment]
Thanks Jaap, no no I'm not looking for a fix everything is working smoothly, I just thought there might have been new changes since the last official release (e.g. I think I saw a commit mentioning dark theme for Windows, which I had once tried to do manually but didn't succeed)

**Comment by bengineerdavis** [comment]
+1 to this. Love using Zimwiki, but I've noticed you and the community have been hard at work adding more fixes and improvements and the last release is more than a year ago (July 2023) -- it would be great for the wider community to receive those changes, too, especially for those that might not be comfortable installing from source.

Again, thank you so much for your work on this project, I trust this app to get the job done for me both in and out of work -- hope to see a new release soon 😄

**Comment by mid-kid** [comment]
The fix I'm looking for is avoiding 100% CPU usage when using Tools->Edit Source on windows. I had built a copy of Zim from git but recent corporate security policy changes made it impossible to run that version.

**Comment by jaap-karssenberg** [comment]
Put releasing top of my list - current guess is that it will be around
Christmas - but my time availability is a bit unpredictable

--- Jaap

**Reply by maclomhair** [reply]
Uhh a Christmas present! :D Thanks a lot for your work again!

**Reply by ZimTom** [reply]
Hi @jaap-karssenberg ,
that's really good news and I very much appreciate "the timing problem" - especially before Christmas.
Also from my side a big thank you for your engagement, as it is purely driven by enthusiasm, and all this also includes  @dehesselle dealing with the Mac version. 
My question in this context:
Is it already clear whether or not this update will trigger an update cycle for the Mac version as well - as it was most of the time in the past ?
Background:
I decided for other reasons to upgrade to MacOS 15.2 after asking for issues with the step to MacOS 15, but as there was no echo (yet) I don't know if there are no issues or whether there are just to few Mac users left or they are busy with other things.
After deliberately deciding pro ZIM and contra other apps in this category because of the compact and efficient approach to the topic I now use it on a day to day basis. Unfortunately since the update I am locked out and had to go back to an old Windows system in order to continue using and feeding my Wiki.
It would be great if I knew a little more about existance and timing of a prospect of an update curing the incompatibility with MacOS 15.

**Reply by dehesselle** [reply]
I'll usually notice new releases fairly quickly and if not, you can ping me.

There's a lot of "under the hood"-work that people don't know about, so it might take a little longer this time.

**Reply by ZimTom** [reply]
René, thanks a lot for your feedback !
Although I am not into software development myself, I know this "under the hood" problem pretty well in some other context. Thus I am patient and grateful to know tha something is on its way.
The main thing for me is not to be forced to change the Wiki platform from ZIM to something else. As an "old white man" I very much prefer the lightweight and pragmatic approach of ZIM :-)

**Comment by dehesselle** [comment]
I believe the initial question has been answered. Code-wise, 0.76.0 is ready and releases are happening and/or are in the works.

**Reply by patc3** [reply]
I don't see any releases? https://zim-wiki.org/downloads.html

**Comment by jaap-karssenberg** [comment]
Check again ;)

**Reply by patc3** [reply]
Thanks, I guess it was my cache 😭😭 Thanks a lot for the update

**Comment by jaap-karssenberg** [comment]
No, I was updating the website right when you were commenting ;)

**Reply by patc3** [reply]
I have a first-row seat to the update😍😍😍

**Comment by ZimTom** [comment]
Thanks a lot Jaap @jaap-karssenberg for this new release and keeping ZIM alive and ...
thanks a lot René @dehesselle for doing all this for the Mac part of the world.
👏😍👏😍 👏😍
